src/igvfd/types/sample.py

```diff
@@ -200,6 +177,98 @@ def taxa(self, request, donors):
         if len(taxas) == 1:
             return list(taxas).pop()
 
+    @calculated_property(
```

Comment:
would not it make sense to have individual sample types of summaries, instead of putting them all together in one big function?

Reply:
In the commit [in vitro system summary update](https://github.com/IGVF-DACC/igvfd/pull/549/commits/4918aed7e84b8900c23e47884958f67f51a00e3c) I had them listed in individual summaries, so if you take a look and really prefer it I can change it back. 

But I really don't think this is better since there is so much redundant code between the summaries and if there were to be further edits to the summaries they would be easily modified here whereas the separate summaries would all need to all be edited with the same changes. Although the summary is longer, the child summaries are just as long, I think this is cleaner.